docs/en/ingest-management/elastic-agent/configuring-kubernetes-metadata.asciidoc

```diff
@@ -0,0 +1,41 @@
+[[configuring-kubernetes-metadata]]
+= Configuring kubernetes metadata enrichment on {agent} managed by {fleet}
+
+When the elastic-agent's policy includes kubernetes integration which configures the collection of kubernetes related metrics
+and container logs the mechanisms used for the metadata enrichment are:
+a. kubernetes provider for log collection
+b. kubernetes metadata enrichers for metrics
+
+In case the elastic-agent's policy does not include kubernetes integration, but {agent} runs inside a kubernetes
+environment the kubernetes metadata are collected by the https://www.elastic.co/guide/en/beats/metricbeat/current/add-kubernetes-metadata.html[add_kubernetes_metadata processor]. The processor is configurable when {agent} is managed by {fleet}.
```

Comment:
NOTE: This document focuses on {agent} configuration solely. In case beats metadata enrichment for kubernetes needed advice relevant filebeat and metricbeat links

Reply:
after the beats prs